frontend/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [main, master]
```

Comment:
Let's definitely remove references to `master` here, but why are we doing a separate YAML file for this? Just checking as there's lots of Playwright stuff in the frontend CI :)

Reply:
this is the example "boilerplate" file... not used in practice I believe.

Reply:
it gets created on `yarn create playwright`